WhaleSpotting/react-app/src/components/Card.tsx

```diff
@@ -20,8 +23,8 @@ export default function Card({ sighting, admin = false }: CardProps): JSX.Elemen
                 <div className="card-info"
                     onClick={() => setCardState(!closeCard)} data-testid="card">
                     <div className="first-column">
-                        <div>Sighted At: {sighting.sightedAt}</div>
-                        <div>Species: {sighting.species}</div>
+                        <div>Sighted At: {sighting.sightedAt.split("T")[0].replaceAll("-", "/")}</div>
+                        <div>Species: {WhaleVisualTextDictionary[Species[sighting.species as keyof typeof Species]]}</div>
```

Comment:
I think we should change the species in the SightingApiModel to be the Species enum, then here it would just be 
```suggestion
                        <div>Species: {WhaleVisualTextDictionary[sighting.species]}</div>
```

Reply:
Done